silkrpc/core/evm_debug.cpp

```diff
@@ -268,7 +269,10 @@ asio::awaitable<std::vector<DebugTrace>> DebugExecutor<WorldState, VM>::execute(
         debug_trace.debug_config = config_;
         auto debug_tracer = std::make_shared<debug::DebugTracer>(debug_trace.debug_logs, config_);
 
-        const auto execution_result = co_await executor.call(block, txn, /* refund */false, /* gasBailout */false, debug_tracer);
+        silkrpc::Tracers tracers;
+        tracers.push_back(debug_tracer);
```

Comment:
Probably better this way: `Tracers debug_tracers{debug_tracer};`

Reply:
Done